keras/src/layers/convolutional/depthwise_conv1d.py

```diff
@@ -92,7 +92,7 @@ class DepthwiseConv1D(BaseDepthwiseConv):
     Example:
 
     >>> x = np.random.rand(4, 10, 12)
-    >>> y = keras.layers.DepthwiseConv1D(3, 3, 2, activation='relu')(x)
+    >>> y = keras.layers.DepthwiseConv1D(kernel_size=3, depth_multiplier=3, strides=2, activation='relu')(x)
```

Comment:
This wouldn't work, it's lacking the filters argument

Reply:
Hi @fchollet ,
[DepthwiseConv1D](https://keras.io/api/layers/convolution_layers/depthwise_convolution1d/) doesn't have filter arguments. If the filter  argument is added then it's showing this error `ValueError: Unrecognized keyword arguments passed to DepthwiseConv1D: {'filters': 16}`